src/components/layouts/Breadcrumbs.tsx

```diff
@@ -0,0 +1,35 @@
+// Breadcrumbs.tsx
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Breadcrumb, Layout } from 'antd';
+import { Link } from 'react-router-dom';
+import { type RootState } from '../../redux/store/store';
+
+const Breadcrumbs: React.FC = () => {
+  const breadcrumbs = useSelector(
+    (state: RootState) => state.breadcrumbs.breadcrumbs
+  );
+  const isAuthenticated = localStorage.getItem('isAuthenticated') === 'true';
+
+  return (
+    <>
+      {isAuthenticated && (
+        <Layout>
+          {breadcrumbs.map((breadcrumb) => (
+            // eslint-disable-next-line react/jsx-key
```

Comment:
Please remove this ts ignore the comment and add a `key` prop to the `Breadcrumb` component. For example:

<Breadcrump key={breadcrumb.name} or use `uuid` node package